rotkehlchen/db/cache.py

```diff
@@ -44,6 +51,7 @@ class DBCacheDynamic(Enum):
     LAST_QUERY_ID = '{location}_{location_name}_{account_id}_last_query_id'
     WITHDRAWALS_TS = 'ethwithdrawalsts_{address}'
     WITHDRAWALS_IDX = 'ethwithdrawalsidx_{address}'
+    EXTRA_INTERNAL_TX = f'{EXTRAINTERNALTXPREFIX}_{{tx_hash}}_{{receiver}}'
```

Comment:
Got a better idea after all.

Make this enum have 2 values. Example for one line.
```suggestion
    EXTRA_INTERNAL_TX = f'{EXTRAINTERNALTXPREFIX}_{{tx_hash}}_{{receiver}}', _simple_callback
```

And then modify the `get_db_key` to use `self.value[0]`
And also add a `@property` like so:

```python
@property 
def deserialize_callback -> Callback[[Any], Any]:
    return self.value[1]
```

And then define the simple callback above and give it to all lines.

```python
T = TypeVar('T')

def _simple_callback(value: T) -> T | None :
    return None if value is None else value
```

And for timestamps define and add this one

```python
def _deserialize_timestamp_from_str(value: str) -> Timestamp:
    return Timestamp(int(value))
```